commit ccd098330f2be230733107c3c6b15e05771f7150
Author: Simon Quigley <tsimonq2@ubuntu.com>
Date:   Sat Jun 10 22:23:08 2017 -0500

    Revert "Fix compile due to change in private API in qtdeclarative"

    This patch should only be in the packaging until qtdeclarative final
    gets out of the NEW queue.

diff --git a/src/location/declarativemaps/qdeclarativegeomap.cpp b/src/location/declarativemaps/qdeclarativegeomap.cpp
index 9f4dae3f..45d84470 100644
--- a/src/location/declarativemaps/qdeclarativegeomap.cpp
+++ b/src/location/declarativemaps/qdeclarativegeomap.cpp
@@ -2139,12 +2139,11 @@ bool QDeclarativeGeoMap::sendMouseEvent(QMouseEvent *event)

 bool QDeclarativeGeoMap::sendTouchEvent(QTouchEvent *event)
 {
-    QQuickPointerDevice *touchDevice = QQuickPointerDevice::touchDevice(event->device());
+    const QQuickPointerDevice *touchDevice = QQuickPointerDevice::touchDevice(event->device());
     const QTouchEvent::TouchPoint &point = event->touchPoints().first();
-    QQuickWindowPrivate *windowPriv = QQuickWindowPrivate::get(window());

-    auto touchPointGrabberItem = [touchDevice, windowPriv](const QTouchEvent::TouchPoint &point) -> QQuickItem* {
-        if (QQuickEventPoint *eventPointer = windowPriv->pointerEventInstance(touchDevice)->pointById(point.id()))
+    auto touchPointGrabberItem = [touchDevice](const QTouchEvent::TouchPoint &point) -> QQuickItem* {
+        if (QQuickEventPoint *eventPointer = touchDevice->pointerEvent()->pointById(point.id()))
             return eventPointer->grabber();
         return nullptr;
     };

